bin/evm

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env ruby
 
-$: << File.expand_path(File.join(File.dirname(__FILE__), '..', 'lib'))
+$: << File.join(File.dirname(__dir__), 'lib')
```

Comment:
Expand `__dir__` to `File.dirname(File.realpath(__FILE__))` to fix the tests/keep compatibility with Ruby <2.0?

Reply:
I opted for the latter one.